src/main/resources/assets/paths/datasetBatch.yaml

```diff
@@ -0,0 +1,27 @@
+get:
+  summary: Get Dataset by Id
+  description: Finds Dataset and DatasetProperties of the Dataset specified by id.
+  parameters:
+    - name: ids
+      in: query
+      description: Comma separate dataset ids, e.g. "1,2,3,4,5"
+      required: true
+      schema:
+        type: array
+        items:
+          type: integer
```

Comment:
The curl call shows a different format: `/api/dataset/batch?ids=2&ids=369`

If, instead, I use a comma separated list this way, Jersey doesn't actually parse the list in the way that is described in the docs:
```
curl -X 'GET' 'https://local.broadinstitute.org:27443/api/dataset/batch?ids=2,369'
```
yields
```
{"code":404,"message":"HTTP 404 Not Found"}
```
since that endpoint doesn't exist